tests/manage/z_cluster/test_hugepages.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import pytest
+
+from ocs_ci.helpers.sanity_helpers import Sanity
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import (
+    enable_huge_pages,
+    disable_huge_pages,
+)
+from ocs_ci.ocs.node import (
+    wait_for_nodes_status,
+    get_nodes,
+)
+from ocs_ci.ocs.resources.pod import (
+    wait_for_pods_to_be_running,
+)
+from ocs_ci.framework.testlib import (
+    bugzilla,
+    skipif_ocs_version,
+    skipif_external_mode,
+    ignore_leftovers,
+    E2ETest,
+    tier2,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@skipif_external_mode
```

Comment:
We can configure hugepages on openshift nodes and then are consumed by apps. I think it should valid even for external mode, any reason for skipping the test to run on external mode?

Reply:
ack, removed skip